### ISSUE TITLE
[suggestion] ocamldep and codept

### ISSUE THREAD

**Issue by ulugbekna** [ocaml-explore/explore#59]
The tools in the title seem to be useful but not famous. Maybe worth mentioning them somewhere on the website?

[codept](https://github.com/Octachron/codept)

Feel free to close if that's not what we want on Explore

**Comment by patricoferris** [comment]
Looks useful! Do you see a [workflow](https://github.com/ocaml-explore/explore/tree/trunk/content/workflows) they would work well with or perhaps new one? Maybe "Navigating an OCaml Project"?

**Comment by ulugbekna** [comment]
> Maybe "Navigating an OCaml Project"

Sounds great! That tool would be indeed very helpful to get around a new codebase.

On a side note: is OCaml Explore about workflows only and just mentioning a tool with its use a realm of ocamlverse? I would be interested to learn the diff between Explore and OCamlverse, in general.

**Comment by patricoferris** [comment]
I think that probably means I should explain it better on the homepage.... 

TL;DR - I like to think of Explore OCaml as a natural successor of Real World OCaml (RWO) but focusing on more than just library authors and still providing common workflows for even beginners still learning the language (like Navigating a Project). 

Explore OCaml is about common OCaml Workflows with a strong dependency on the [OCaml Platform](https://www.youtube.com/watch?v=E8T_4zqWmq8) -- a suite of tools that are/were/will be pivotal to being productive in OCaml for example Dune, OCamlformat or Dune-Release.

Explore OCaml focuses on these **workflows** and how they link to the **platform tools** and how they link to our (currently) six **users**: beginners, teachers, library authors, applications devs, end-user and distro managers. It tries to write the workflows that RWO does not from a productivity point-of-view. RWO is for learning the OCaml language. OCamlverse is a great index of **many different** aspects of the OCaml Universe. 

Of course there will be tools that are outside of the platform and these are somewhat dubbed "community libraries". For example this a workflow for running code in the browser that uses dune and `js_of_ocaml`. 

Importantly Explore OCaml seeks to be up to date -- it uses dune and mdx to make sure all of the workflows and examples are running. The CLI tool checks for outdated workflows too in order to stay as up to date as possible.

I hope that is somewhat clearer haha :))
